app/components/Image/index.tsx

```diff
@@ -7,11 +7,13 @@ import './index.scss';
 
 type ImageProps = {
   image?: ImageType;
-  height?: number;
-  width?: number;
+  height?: number | string;
+  width?: number | string;
   showTitle?: boolean;
   showCaption?: boolean;
   isZoomable?: boolean;
+  /** Props to the img tag directly */
+  imgProps?: React.ComponentPropsWithoutRef<'img'>;
```

Comment:
I think it's confusing to call this `imgProps` when a lot of the other props also end up directly on the `img` tag (ex. width, height etc). Since what you actually want is just the maxHeight, I would just add that as the prop. or if you know there's other styles you're going to be adding maybe `imgStyles` as the prop

Reply:
actually since the calculation for the `maxHeight` in the 2 cases where it's applied is the same, maybe a prop of like 
`shrinkToFitViewport={true}` might be better still

Reply:
since there are other places where the `Image` component is used, my thinking is that if the normal height/width prop is not enough to style the component, then this prop would provide the override. Sure this is an encompass-all prop, but MUI does the same escape hatch thing as well, and if we do need more stuff in the future that's going to be more changes to the prop which IMO is just extra work 

`shrinkToFitViewport={true}` could be a prop, but it'd probably needs to specify `vw` or `vh` ...?

Reply:
`imgStyles` would probably be our best bet here,

Reply:
[923f6f4](https://github.com/bcgsc/pori_ipr_client/pull/187/commits/923f6f4402e53ceeb1fafcd75be8f833a7cc5868)

Reply:
> shrinkToFitViewport={true} could be a prop, but it'd probably needs to specify vw or vh ...?

in both cases where you're setting maxHeight the code is identical so there would be no need to supply a value, just move that code into the Image component instead and apply conditionally